crates/ruff_server/docs/setup/NEOVIM.md

```diff
@@ -0,0 +1,52 @@
+## NeoVim Setup Guide for `ruff server`
+
+### Using `nvim-lspconfig`
+
+1. Install [`nvim-lspconfig`](https://github.com/neovim/nvim-lspconfig).
+1. Setup `nvim-lspconfig` with the [suggested configuration](https://github.com/neovim/nvim-lspconfig/tree/master#suggested-configuration).
+1. Finally, add this to your `init.lua`:
+
+```lua
+require'lspconfig'.ruff.setup{}
+```
+
+See [`nvim-lspconfig`'s server configuration guide](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#ruff) for more details
+on how to configure the server from there.
+
+**Important note**: If you have the older language server (`ruff-lsp`) configured in NeoVim, make sure to disable it to prevent any conflicts.
```

Comment:
We can format it even better :)

```suggestion
> [!NOTE]
>
> If you have the older language server (`ruff-lsp`) configured in NeoVim, make sure to disable it to prevent any conflicts.
```

<img width="1214" alt="Screenshot 2024-04-17 at 08 04 57" src="https://github.com/astral-sh/ruff/assets/67177269/6fe7f2a5-74fb-4ba2-ba27-ba7c58ecc096">

Reply:
There are others as well: `[!IMPORTANT]`, `[!WARNING]`, `[!TIP]`. I guess for this we can either go with "important" or "note".